src/dataflow/src/source/mod.rs

```diff
@@ -1474,6 +1521,26 @@ where
                 }
             };
 
+            if bindings_cap.time() == &0 {
```

Comment:
this is triggering on the capability being zero but is the intention really: please run this once before anything else happens? just want to double check

Reply:
Yes, that is correct. I don't like it, but I didn't know how else to do it. I could do it if I rewrote the source code to use a custom operator, where we can just emit a thing immediately when starting up. But that seems more complicated.